base/broadcast.jl

```diff
@@ -1242,7 +1242,31 @@ macro __dot__(x)
     esc(__dot__(x))
 end
 
-@inline broadcasted_kwsyntax(f, args...; kwargs...) = broadcasted((args...)->f(args...; kwargs...), args...)
+"""
+    Base.Broadcast.FixKwargs{F,K} <: Function
```

Comment:
Is it better to move this to `base/operators.jl` where `Fix1` and `Fix2` live?

Reply:
Yes, I like that idea.